dask_ec2/formulas/salt/dask/distributed/init.sls

```diff
@@ -4,24 +4,21 @@ include:
   - conda
   - system.base
 
-dask-install:
-  pip.installed:
-    - name: dask
-    - bin_env: {{ install_prefix }}/bin/pip
+bokeh-install:
+  cmd.run:
+    - name: {{ install_prefix }}/bin/conda install bokeh -y -q
     - require:
       - sls: conda
 
-bokeh-install:
+dask-install:
   cmd.run:
-    - name: {{ install_prefix }}/bin/conda install bokeh -y -q
+    - name: {{ install_prefix }}/bin/conda install dask -y -q -c conda-forge
     - require:
       - sls: conda
 
 distributed-install:
-  pip.installed:
-    - name: distributed
-    - upgrade: true
-    - bin_env: {{ install_prefix }}/bin/pip
+  cmd.run:
+    - name: {{ install_prefix }}/bin/conda install distributed -y -q -c conda-forge
```

Comment:
No disagreement, although changing this might require me to understand how salt works.  All of the changes that I've done here are more-or-less just renamings.  I haven't yet had to change any of the actual structure of the specification.